src/api/c/stats.h

```diff
@@ -40,39 +40,52 @@ struct baseOutType {
                                 float>::type type;
 };
 
-template<typename T>
-inline T mean(const Array<T>& in)
+template<typename Ti, typename To>
+inline To mean(const Array<Ti>& in)
```

Comment:
I think you will have to make these statics. That's why windows is failing I believe.